mmv1/products/identityplatform/ProjectDefaultConfig.yaml

```diff
@@ -124,3 +124,136 @@ properties:
             description: |
               Memory cost for hash calculation. Used by scrypt and other similar password derivation algorithms. See https://tools.ietf.org/html/rfc7914 for explanation of field.
+
+  - !ruby/object:Api::Type::NestedObject
+    name: 'blockingFunctions'
+    description: |
+      Configuration related to blocking functions.
+    properties:
+      - !ruby/object:Api::Type::Map
+        name: 'triggers'
+        description: |
+          Map of Trigger to event type. Key should be one of the supported event types: "beforeCreate", "beforeSignIn".
+        value_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'functionUri'
+              description: |
+                HTTP URI trigger for the Cloud Function.
+            - !ruby/object:Api::Type::NestedObject
+              name: 'updateTime'
+              description: |
+                When the trigger was changed.
+              properties:
+                - !ruby/object:Api::Type::Integer
+                  name: 'seconds'
+                - !ruby/object:Api::Type::Integer
+                  name: 'nanos'
+      - !ruby/object:Api::Type::NestedObject
+        name: 'forwardInboundCredentials'
+        description: |
+          The user credentials to include in the JWT payload that is sent to the registered Blocking Functions.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'idToken'
+            description: |
+              Whether to pass the user's OIDC identity provider's ID token.
+          - !ruby/object:Api::Type::Boolean
+            name: 'accessToken'
+            description: |
+              Whether to pass the user's OAuth identity provider's access token.
+          - !ruby/object:Api::Type::Boolean
+            name: 'refreshToken'
+            description: |
+              Whether to pass the user's OAuth identity provider's refresh token.
+ 
+  - !ruby/object:Api::Type::NestedObject
+    name: 'quota'
+    description: |
+      Configuration related to quotas.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'signUpQuotaConfig'
+        description: |
+          Quota for the Signup endpoint, if overwritten. Signup quota is measured in sign ups per project per hour per IP.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'quota'
+            description: |
+              Corresponds to the 'refill_token_count' field in QuotaServer config.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'startTime'
+            description: |
+              When this quota will take affect.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'seconds'
+              - !ruby/object:Api::Type::Integer
+                name: 'nanos'
+          - !ruby/object:Api::Type::NestedObject
+            name: 'quotaDuration'
```

Comment:
This is a string based on https://cloud.google.com/identity-platform/docs/reference/rest/v2/Config#TemporaryQuota